e2e/cypress/integration/system_console/search_box_spec.js

```diff
@@ -18,11 +18,10 @@ const goToAdminConsole = () => {
 };
 
 describe('System console', () => {
-    before(() => {
+    it('MM-T897_1 - Focus should be in System Console search box on opening System Console or refreshing pages in System Console', () => {
+        // * Check if server has license
         cy.apiRequireLicense();
-    });
```

Comment:
This step is only executed on the first testcase. The last test `MM-T1634 - Search box should remain visible / in the header as you scroll down the settings list in the left-hand-side` is also dependent on EE license.

Reply:
I added a `apiDeleteLicense` locally on `MM-T1634` and it doesn't seem to require a license. Does it fail for you when you add `apiDeleteLicense`

Reply:
Sorry. My apologies. I thought Bleve Indexing is only available for EE. Just found that its also available for TE. Please ignore.